superset/assets/javascripts/components/StackTraceMessage.jsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Collapse } from 'react-bootstrap';
+
+const propTypes = {
+  message: PropTypes.string,
+  queryResponse: PropTypes.object,
+  showStackTrace: PropTypes.bool,
+  removeAlert: PropTypes.func,
+};
+const defaultProps = {
+  showStackTrace: false,
+};
+
+class StackTraceMessage extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hidden: false,
+      showStackTrace: props.showStackTrace,
+    };
+  }
+
+  removeAlert() {
```

Comment:
I think the feature allowing to remove alerts is kind of legacy (not sure how it got to be a requirement in the first place). I think ExploreV2 was trying to replicate an MVC legacy feature. Anyhow we could remove that (and related actions/reducers) as part of this PR or later if you prefer. I'm just putting this on the radar here.

Reply:
fixed. removed removeAlerts feature.